### PR TITLE
Fix xkb layouts saved as ibus sources

### PR DESCRIPTION
**Pull request by jeremypw**
Fixes https://github.com/elementary/wingpanel-indicator-keyboard/issues/80

* Move to InputSources directory
* Rename to help distinguish input source and xkb layout
* Some simplification
* Ensure Object () first statement in constructor
* Fix some long lines

The cause of the issue not yet identified but after this refactoring I have not been able to reproduce it again.

I think it maybe something to do with how some Layout object constructors did not have Object () as the first line.

NOTE: You should log out or restart the wingpanel after installing this in order for the wingpanel indicator to show the ibus methods correctly you may also need to run the plug to reconstruct the settings correctly.

### CODE REVIEW

File: src/Widgets/Layout/Display.vala [hunk 1]
```diff
@@ -18,8 +18,9 @@
 */
 
 // widget to display/add/remove/move keyboard layouts
-public class Pantheon.Keyboard.LayoutPage.Display : Gtk.Frame {
-    private LayoutSettings settings;
+namespace Pantheon.Keyboard {
+public class LayoutPage.Display : Gtk.Frame {
```
Comment (ryonakano) [inline src/Widgets/Layout/Display.vala:22]: We might want to add indentation from L22 to L198.

**Review by ryonakano — COMMENTED** [review]
Confirmed the issue fixes.

File: src/Widgets/Layout/Display.vala (hunk 1 above)
Comment (jeremypw) [reply src/Widgets/Layout/Display.vala]: I can't remember why I split out the namespace now ... I have reverted that and it still compiles.  Thanks for the review.

**Comment by tintou** [comment]
Don't forget to update the POTFILES if you rename some files

**Review by ryonakano — CHANGES_REQUESTED** [review]
There seems to be a regression that you can't add variants of a layout, e.g. if you already set up jp keyboard and try to adding jp+dvorak, nothing happens.

**Review by ryonakano — APPROVED** [review]
This fixes the linked issue and no regressions found. Thank you! :tada:

**Pull request merged**